.travis.yml

```diff
@@ -9,3 +9,4 @@ script:
 
 after_success:
   - bash <(curl -s https://codecov.io/bash)
+  - if [ -n "$TRAVIS_TAG" && "$TRAVIS_GO_VERSION" == "1.8" ]; then ./.ci-docker.sh; fi
```

Comment:
You may use `[ -n "$TRAVIS_TAG" && "$TRAVIS_GO_VERSION" == "1.8" ] && ./.ci-docker.sh`